mit_frontend/package-lock.json

```diff
@@ -0,0 +1,9692 @@
+{
```

Comment:
Remove this file from the commit.

Reply:
Actually ignore this particular comment, as described in [npm](https://docs.npmjs.com/files/package-lock.json) this file is intended to be submitted to repository to make sure everyone's using the same dependency tree.